README.md

```diff
@@ -249,10 +249,7 @@ Controller controller = Get.put(Controller()); // Rather Controller controller =
 ```
 
 - Note: If you are using Get's State Manager, pay more attention to the bindings api, which will make easier to connect your view to your controller.
-
-Instead of instantiating your class within the class you are using, you are instantiating it within the Get instance, which will make it available throughout your App.
-So you can use your controller (or class Bloc) normally
-
+https://github.com/jonataslaw/ge
```

Comment:
Hi, I think this link was accidentally pasted here.

Reply:
Yes indeed